ember-cli-build.js

```diff
@@ -3,7 +3,7 @@
 var path = require('path');
 var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 var EmberApp = require('ember-cli/lib/broccoli/ember-app');
-var shim = require('./lib/pod-templates-shim');
+var shim = require('flexi-layouts/lib/pod-templates-shim');
```

Comment:
Derp. Forgot this is needed for apps with a pod structure, and that this only applies to the dummy app